shared/engine/index.js

```diff
@@ -154,14 +157,14 @@ class Engine {
       result: (...args) => {
         if (once) {
           if (printRPC) {
-            console.log('RPC ▼ result bailing on additional calls: ', method, param, ...args)
+            console.logLocal('RPC ▼ result bailing on additional calls: ', method, param, ...args)
```

Comment:
i think we should avoid enhancing console with our custom stuff. us monkeypatching remote components to route the output i think is ok but i don't want a bunch of random methods in there that are new